format_checker/common_checks.py

```diff
@@ -23,6 +24,27 @@
 }
 
 
+def check_repo_sanity(checked_projects, filename, row, i, log):
+    project_url = row["Project URL"]
+    if project_url in checked_projects:
+        return
+    checked_projects.add(project_url)
+
+    match = common_data["Project URL"].fullmatch(project_url)
+    author = match.group(2)
+    repo = match.group(3)
+
+    url = "https://api.github.com/repos/{}/{}".format(author, repo)
+    try:
+        resp = requests.get(url).json()
+        # Determine if it is a forked project
+        if resp.get("fork"):
+            log_esp_error(filename, log, f"{author}/{repo} is a forked repo")
+    except requests.exceptions.RequestException as e:
+        # handle(e)
+        pass
```

Comment:
Instead of simply passing here, it would be good to warn that the check for fork failed along with outputting the exception